test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'cli.js'
+);
+
+const args = [
+  npmPath,
+  'install'
+];
+
+let stdout = '';
+let stderr = '';
+
+const proc = spawn(process.execPath, args, {
+  stdio: 'pipe',
+  cwd: common.tmpDir
+});
+
+proc.stdout.setEncoding('utf8');
+proc.stderr.setEncoding('utf8');
+
+proc.stdout.on('data', (data) => stdout += data);
+proc.stdout.on('err', (data) => stderr += data);
+
+proc.on('exit', function(code) {
+  // npm install in tmpDir should be essentially a no-op
+  // if npm is not broken the process should always exit 0
+  // assert that there are no obvious failures
```

Comment:
Nit: capitalization and (more importantly) punctuation in comments. I tried reading the first two lines four or five times before I figured out that each line is its own sentence.